### ISSUE TITLE
[BUG] Phone Number Auth Error Displayed along with Custom Message

### ISSUE THREAD

**Issue by CorporateInvesther** [reach4help/reach4help#590]
https://reach4help-staging-web-client.firebaseapp.com/phone/entry

**Describe the bug**
When an auth error is triggered, the error message is displayed alongside the custom message. 

Not enough digits in phone number: 

![TOO_SHORT](https://user-images.githubusercontent.com/65609627/82918278-7a7ea700-9f42-11ea-97f0-db3acefa061a.PNG)

Too many digits in phone number: 

![TOO LONG](https://user-images.githubusercontent.com/65609627/82918389-a26e0a80-9f42-11ea-97bb-bd9712427320.PNG)

**To Reproduce**
1. Go to /phone/entry
2. Enter an invalid phone number
3. Notice the error message 

**Expected behavior**
User should not be seeing raw errors like TOO_SHORT. The message should be displayed in laymans terms

**Screenshots**
Screenshots above. 

**Desktop (please complete the following information):**
Windows - Chrome - (incognito mode)

--------
### [ZenHub Information](https://app.zenhub.com/workspaces/reach4help-5e8dcbfb14ac087f410cbabb/issues/reach4help/reach4help/590)

*This information is updated automatically. To modify it, please use ZenHub.*

**Comment by djohal** [comment]
This was addressed with #587

**Comment by isma-osy** [comment]
Categorized as MAJOR

**Comment by CorporateInvesther** [comment]
Verified Fixed v2.1.0
